Sources/TLS/Socket.swift

```diff
@@ -9,6 +9,7 @@ public final class Stream {
     public let cConfig: CConfig
     public let context: Context
     public let certificates: Certificates
+    public let config: TLSConfig
```

Comment:
This should just be called `Config` since it's already namespaced under the tLS module.